updateOrDeploy.sh

```diff
@@ -23,6 +23,7 @@ if [ -d ~ ]
 then
     dirs+=" $HOME/.IntelliJIdea"
     dirs+=" $HOME/.IdeaIC"
+    dirs+=" $HOME/.IdeaIC14"
```

Comment:
I have no idea. We're at IDEA 14.1 now. I guess that since they decided to use .IdeaIC14 now, probably a future version will use .IdeaIC15, so you can have both versions installed at the same time.